tidy3d/components/source.py

```diff
@@ -180,10 +187,10 @@ def frequency_range(self) -> FreqBound:
         Tuple[float, float]
             Minimum and maximum frequencies of the
             :class:`GaussianPulse` or :class:`ContinuousWave` power
-            within 4 standard deviations.
+            within WIDTH_STD standard deviations.
```

Comment:
hm, I wonder if there's a way to specify what this in case people want to know.  Maybe we should make width_f a kwarg with default value `WIDTH_STD`?